editor/plugins/animation_state_machine_editor.cpp

```diff
@@ -1565,6 +1561,7 @@ void AnimationNodeStateMachineEditor::_notification(int p_what) {
 			transition_mode->add_icon_item(get_theme_icon(SNAME("TransitionImmediate"), SNAME("EditorIcons")), TTR("Immediate"));
 			transition_mode->add_icon_item(get_theme_icon(SNAME("TransitionSync"), SNAME("EditorIcons")), TTR("Sync"));
 			transition_mode->add_icon_item(get_theme_icon(SNAME("TransitionEnd"), SNAME("EditorIcons")), TTR("At End"));
+			transition_mode->add_icon_item(get_theme_icon(SNAME("TransitionTime"), SNAME("EditorIcons")), TTR("At Time"));
```

Comment:
Seems like an unrelated change that crept up from other work? I don't see `SWITCH_MODE_AT_TIME` in `AnimationNodeStateMachineTransition::SwitchMode`.

Reply:
Yes, that got accidentally committed from some unrelated experimental work I was doing. I shall remove it now 👀